packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -1286,25 +1294,29 @@ function internalLoadAssetContainerAsync(
     pluginOptions?: PluginOptions
 ): Promise<AssetContainer> {
     return new Promise((resolve, reject) => {
-        loadAssetContainerCoreAsync(
-            rootUrl,
-            sceneFilename,
-            scene,
-            (assets) => {
-                resolve(assets);
-            },
-            onProgress,
-            (scene, message, exception) => {
-                reject(exception || new Error(message));
-            },
-            pluginExtension,
-            name,
-            pluginOptions
-        );
+        try {
+            loadAssetContainerCoreAsync(
+                rootUrl,
+                sceneFilename,
+                scene,
+                (assets) => {
+                    resolve(assets);
+                },
+                onProgress,
+                (scene, message, exception) => {
+                    reject(exception || new Error(message));
+                },
+                pluginExtension,
+                name,
+                pluginOptions
+            ).catch((error) => reject(error));
+        } catch (error) {
+            reject(error);
+        }
     });
 }
 
-function importAnimations(
+async function importAnimations(

```

Comment:
as they are internal, can you please rename all the new async with Async at the end?

Reply:
I can, but there are already other variations of these functions that have the `Async` postfix. For example, there is already a `importAnimationsAsync`, so I would need to then rename that one as well. Because of this, and the fact that I anticipate this being heavily rewritten in the nearish future, I opted not to try to improve the naming for the internal functions. If you feel strongly about it though, I could try to come up with some short term names that make more sense.

Reply:
they are internal so maybe _importAnimationsAsync ?

Reply:
Ok, I did a pass to make the function names more consistent, use the `Async` postfix, and added a few comments to make it more clear what the purpose of each internal function is.